07.ls_object/ls.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require_relative './file_display'
+require_relative './file_detail'
+require_relative './file_mode'
+
+FileDisplay.new.show_formatted_file_details
```

Comment:
```suggestion
FileDisplay.new.show_file_details
```

とか

```suggestion
FileDisplay.new.show_files
```

とかでもいいですね。
理由としては「わざわざ詳しく説明する必要がないから」です。

https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name

Reply:
承知しました！
他の変数、メソッドと名前を区別する必要がない場合は、シンプルな命名でよいということですね。
こちら修正しました。